compose.yaml

```diff
@@ -6,11 +6,11 @@ services:
     container_name: ovh-exporter
     ports:
       - "8080:8080"
-    environment:
-      OVH_ENDPOINT: "ovh-eu"
-      OVH_APP_KEY: ""
-      OVH_APP_SECRET: ""
-      OVH_CONSUMER_KEY: ""
-      OVH_CLOUD_PROJECT_INSTANCE_BILLING_PROJECT_IDS: ""
-      OVH_CACHE_UPDATE_INTERVAL: "300"
-      SERVER_PORT: "8080"
+    env_file: ".env"
+      # OVH_ENDPOINT: "ovh-eu"
```

Comment:
you can remove these since we explain what should be contained inside the ovh-exporter.env file